runelite-client/src/main/java/net/runelite/client/plugins/boosts/BoostsPlugin.java

```diff
@@ -224,24 +321,52 @@ public void addStatChangeIndicator()
 	 *
 	 * @return integer value in seconds until next boost change
 	 */
-	public int getChangeTime()
+	private int getChangeDownTicks()
 	{
-		int timeSinceChange = timeSinceLastChange();
+		if (lastChangeDown == -1 || (config.displayNextBuffChange() == BoostsConfig.DisplayChangeMode.BOOSTED && !isChangedUp))
+		{
+			return -1;
+		}
+
+		int ticksSinceChange = client.getTickCount() - lastChangeDown;
 		boolean isPreserveActive = client.isPrayerActive(Prayer.PRESERVE);
 
-		if ((isPreserveActive && (timeSinceChange < 45 || preserveBeenActive)) || timeSinceChange > 75)
+		if ((isPreserveActive && (ticksSinceChange < 75 || preserveBeenActive)) || ticksSinceChange > 125)
 		{
 			preserveBeenActive = true;
-			return 90 - timeSinceChange;
+			return 150 - ticksSinceChange;
 		}
 
 		preserveBeenActive = false;
-		return (timeSinceChange > 60) ? 75 - timeSinceChange : 60 - timeSinceChange;
+		return (ticksSinceChange > 100) ? 125 - ticksSinceChange : 100 - ticksSinceChange;
 	}
 
-	private int timeSinceLastChange()
+	int getChangeDownTime()
 	{
-		return (int) Duration.between(lastChange, Instant.now()).getSeconds();
+		final int changeDownTicks = getChangeDownTicks();
+		return changeDownTicks != -1 ? (int)(changeDownTicks * 0.6) : changeDownTicks;
```

Comment:
This should probably account for the time between call and when it the last tick was so the timer decreases at a regular cadence.